api/rtx/src/uvisor_malloc.c

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright (c) 2016, ARM Limited, All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <stdint.h>
+#include <stddef.h>
+#include <stdio.h>
+#include <reent.h>
+
+#include "uvisor-lib/uvisor-lib.h"
+#include "cmsis_os.h"
+
+#define DPRINTF(...) {};
+/* Use printf with caution inside malloc: printf may allocate memory itself,
```

Comment:
Please add this comment also [here](https://github.com/ARMmbed/uvisor/pull/231/commits/6344d477be04aeecf15bcce9ab781710360a41bb#r65918140).